automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/cluster/ClusterConcurrencySpec.scala

```diff
@@ -1,144 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo.cluster
-
-import org.broadinstitute.dsde.workbench.fixture.BillingFixtures
-import org.broadinstitute.dsde.workbench.leonardo.Leonardo.ApiVersion.V2
-import org.broadinstitute.dsde.workbench.leonardo.{ClusterStatus, Leonardo, LeonardoTestUtils}
-import org.broadinstitute.dsde.workbench.service.util.Tags
-import org.scalatest.time.{Seconds, Span}
-import org.scalatest.{FreeSpec, ParallelTestExecution}
-
-
-/**
-  * Created by rtitle on 4/26/18.
-  */
-class ClusterConcurrencySpec extends FreeSpec with LeonardoTestUtils with ParallelTestExecution with BillingFixtures {
```

Comment:
I removed this spec entirely. It was hard to read, error-prone, and took a long time. It's replaced with `ClusterStatusTransitionsSpec` which I believe covers essentially the same things.